src/amo/components/AddonBadges/index.js

```diff
@@ -50,6 +50,12 @@ export const AddonBadgesBase = (props: Props) => {
           label={i18n.gettext('Experimental')}
         />
       ) : null}
+      {addon && !addon.isWebExtension ? (
```

Comment:
What if the extension is:

* a theme
* a language tool
* a search plugin

Will it be marked as incompatible then?

Reply:
oh shoot, that's right. I totally forgot this check :|